.github/workflows/backend-tests.yml

```diff
@@ -70,8 +70,45 @@ jobs:
         working-directory: .
         run: ./scripts/test.backend.sh
 
+  format:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-backend-tests-${{ hashFiles('Cargo.lock', 'rust-toolchain.toml') }}
+      - name: Install cargo dependency sorter
+        run: cargo install cargo-sort@1.0.9
+      - name: Format
+        run: ./scripts/format.sh
+      - name: Check formatted
+        run: |
+          test -z "$(git status --porcelain)" || {
+                  echo "FIX: Please run ./scripts/format.sh"
+                  git diff
+                  exit 1
+          }
+
+  lint:
```

Comment:
Can also be moved to `backend-checks.yml`

Reply:
done